src/domains/wallet/components/WalletsList/WalletsList.tsx

```diff
@@ -174,7 +174,7 @@ export const WalletsList: React.VFC<WalletsListProperties> = ({
 								rowsPerPage={itemsPerPage}
 								currentPage={currentPage}
 								initialState={initialState}
-								className="with-x-padding"
+								className="with-x-padding after:mx-[0.15rem] after:block after:h-[5px] after:rounded-b-lg after:bg-theme-primary-100 after:content-[''] after:dark:bg-theme-secondary-800"
```

Comment:
Have you tried this trick for the border bottom? https://github.com/ArdentHQ/arkvault/blob/c7f12e38a357c8caf12134e462a10a3c766a990d/src/domains/vote/components/AddressTable/AddressTable.tsx#L153

Reply:
@goga-m Am I missing something?

![image](https://github.com/user-attachments/assets/a39f7472-0c60-4962-81cb-476f4fa279ca)